packages/dd-trace/src/appsec/iast/taint-tracking/operations.js

```diff
@@ -128,7 +139,7 @@ function enableTaintOperations (telemetryVerbosity) {
     onRemoveTransaction = onRemoveTransactionInformationTelemetry
   }
 
-  global._ddiast = getTaintTrackingImpl(telemetryVerbosity)
+  global._ddiast = getTaintTrackingImpl(telemetryVerbosity, taintObject, false)
```

Comment:
in order to avoid a cyclic reference I have to pass `taintObject` method as an argument here